### PR TITLE
remove white flash screen on startup

### PR DESCRIPTION
**Pull request by estebanmino**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)

removing flash white screen on startup

with this pr we'll be initializing theme from index.html

before we were doing this from `initThemingCritical` and `initThemingBody` causing the flash since those take a fraction of a second after dom is loaded to be initialized

## Screen recordings / screenshots

https://www.loom.com/share/f0e221cff41b4f188cc00462cad61fd2

### CODE REVIEW

File: src/design-system/styles/initThemingLocal.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { currentThemeStore } from '~/core/state/currentSettings/currentTheme';
+
+export function initThemingLocal() {
+  const localTheme = localStorage.getItem('theme');
+  if (!localTheme) {
+    const currentTheme = currentThemeStore.getState().currentTheme;
```
Comment (greg-schrammel) [inline src/design-system/styles/initThemingLocal.ts:6]: is the extra localStorage set/get needed? can't just use the zustand store like this?

Comment (derHowie) [reply src/design-system/styles/initThemingLocal.ts:6]: guessing this is so the static vanilla js scripts have somewhere to read from

**Review by derHowie — APPROVED** [review]
🙏

File: src/design-system/styles/initThemingLocal.ts (hunk 1 above)
Comment (estebanmino) [reply src/design-system/styles/initThemingLocal.ts:6]: yeah exactly what @derHowie says

Comment (greg-schrammel) [reply src/design-system/styles/initThemingLocal.ts:6]: aah zustand is using an async `chrome.storage.local` for everything, maybe we could consider opting for standard `localStorage` when fit
so the store is already hydrated when created vs having to wait for the async run to hydrate it

**Pull request merged**